README.md

```diff
@@ -34,7 +34,7 @@ The following command-line utilties must be present in the `PATH`:
 * the filesystem listed as `-default-fs` (defaults to: `xfs`)
 
 This plugin's tests are run in a centos 7.3.1611 container with
-lvm2-2.02.171-8.el7 installed. It should work with newer versions of lvm2 that
+lvm2-2.02.177-8.el7 installed. It should work with newer versions of lvm2 that
```

Comment:
if this package exists, then why build from source?

Reply:
The version 2.02.177 (latest) does not exist as a package, only the older one. I'd like to bring the version in the Dockerfile up to date with the latest LVM2 release.